executor/common_usb.h

```diff
@@ -285,6 +289,459 @@ static int configure_device(int fd)
 	return 0;
 }
 
+#if USB_DEBUG
+
+#include <linux/hid.h>
+#include <linux/usb/audio.h>
+#include <linux/usb/cdc.h>
+#include <linux/usb/ch11.h>
+#include <linux/usb/ch9.h>
+
+#include <errno.h>
+#include <fcntl.h>
+#include <stdarg.h>
+#include <stdbool.h>
+#include <string.h>
+#include <sys/stat.h>
+#include <sys/types.h>
+
+// Can't use the common write_file() here, since its definition should
+// depend on USB_DEBUG, which is only defined here.
```

Comment:
USB_DEBUG does not seem to be defined anywhere around. I fit's supposed to be defined on command line, then it should be fine to use for write_file.

Reply:
It's defined at the top of `common_usb.h`. Defining it from the command line would work for me. There's no way currently to pass flags to `make executor`, right? Should we add some way to do that?

Reply:
Nevermind, `CFLAGS="-DUSB_DEBUG=1" make executor` works.

Reply:
Removed `usb_write_file()`, PTAL.